ShittyLINQ/IsNotNullOrEmpty.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using System.Linq;
```

Comment:
I think @jreina was trying to avoid any dependencies on `LINQ`.

Reply:
@the-pat this is correct.

Reply:
@the-pat  & @jreina Apologies, I missed that